firebase-appdistribution/test-app/src/main/java/com/googletest/firebase/appdistribution/testapp/MainActivity.kt

```diff
@@ -264,4 +307,12 @@ class MainActivity : AppCompatActivity() {
         updateAppButton.visibility = View.GONE
         updateAppButtonBackground.visibility = View.GONE
     }
+
+    companion object {
+        const val TAG = "MainActivity"
+        const val TRIGGER_NONE = "None"
+        const val TRIGGER_SHAKE = "Shake the device"
+        const val TRIGGER_SCREENSHOT = "Take a screenshot"
+        const val TRIGGER_NOTIFICATION = "Click the notification"
```

Comment:
I think the four triggers should probably be an enum

Reply:
I thought that added a little too much boilerplate for my liking: having to declare a String field in the enum class, and then access that field everywhere we want to use it in code above. But I gave it a shot and it's actually not bad. Defining the field is super clean. So I like it after all, thanks for the push!